aws-test/tests/aws_rds_db_cluster_snapshot/test-get-expected.json

```diff
@@ -3,8 +3,8 @@
     "arn": "{{ output.resource_aka.value }}",
     "db_cluster_identifier": "{{resourceName}}",
     "db_cluster_snapshot_identifier": "{{resourceName}}",
-    "engine": "aurora",
-    "engine_version": "5.6.mysql_aurora.1.22.5",
+    "engine": "aurora-mysql",
+    "engine_version": "5.7.mysql_aurora.2.11.2",
```

Comment:
Since these output parameters will keep on varying depending on the terraform and the provider version, I feel it is better to remove these parameters from the expected.json file.